src/store/app.ts

```diff
@@ -134,3 +143,22 @@ export const walletUiHrefAtom = atom(get => {
 
   return bridgeUrl ? bridgeUrl.origin + '/wallet/' : '';
 });
+
+// Increment whenever the terms on https://docs.inter.trade/disclaimer change.
+export const latestTermsIndex = 0;
```

Comment:
"Terms" has a meaning in our contract API. consider "Disclaimer" or "[ToS](https://en.wikipedia.org/wiki/Terms_of_service)"